UMLEditor/UMLEditor/Classes/Diagram.cs

```diff
@@ -146,6 +146,37 @@ public void AddRelationship(string sourceClassName, string destClassName, string
         
     }
 
+    /// <summary>
+    /// Adds a method to a class
```

Comment:
Might want to add a mention to what kind of add method this will be, for example "add method to a class with no specified params"